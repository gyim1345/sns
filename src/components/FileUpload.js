import React, { Fragment, useState } from "react";
import Message from "./Message";
import axios from "axios";
import { css } from "@emotion/core";

const FileUpload = ({ currentUser, posting, setPosting }) => {
  const [file, setFile] = useState("");
  const [filename, setFilename] = useState("Choose File");
  const [message, setMessage] = useState("");
  const [imgURL, setImgURL] = useState("");
  const [input, setInput] = useState("");
  const [inputTag, setInputTag] = useState("");
  const onImageChange = event => {
    if (event.target.files && event.target.files[0]) {
      setImgURL(URL.createObjectURL(event.target.files[0]));
    }
  };

  const onChangeInput = e => {
    setInput(e.target.value);
  };

  const onChangeTagInput = e => {
    setInputTag(e.target.value);
  };

  const onChange = e => {
    onImageChange(e);
    setFile(e.target.files[0]);
    setFilename(e.target.files[0].name);
  };
  console.log(file);

  const onSubmit = async e => {
    e.preventDefault();
    const formData = new FormData();
    formData.append("file", file);
    formData.append("input", input);
    formData.append("inputTag", inputTag);
    formData.append("user", currentUser);

    try {
      console.log(formData);
      const { data } = await axios.post(
        "http://localhost:3000/upload",
        formData,
        {
          headers: {
            "Content-Type": "multipart/form-data"
          }
        }
      );
      setPosting([...posting, data.posts]);
      setMessage("File Uploaded");
    } catch (err) {
      if (err.response.status === 500) {
        setMessage("There was a problem with the server");
      } else {
        setMessage(err.response.data.msg);
      }
    }
  };


  return (
    <Fragment>
      {message ? <Message msg={message} /> : null}
      <form onSubmit={onSubmit}>
        <div className="custom-file mb-4">
          <input
            type="file"
            className="custom-file-input"
            id="customFile"
            onChange={onChange}
          />
        </div>

        {imgURL !== "" && (
          <img
            id="target"
            src={imgURL}
            style={{ maxWidth: "500px", maxHeight: "500px" }}
          />
        )}
        <div css={[inputInputTagUpload]}>
          <input type="text" onChange={onChangeInput} placeholder="Title" />
          <input type="text" onChange={onChangeTagInput} placeholder="Tag" />
          <input
            type="submit"
            value="Upload"
            className="btn btn-primary btn-block mt-4"
          />
        </div>
      </form>
    </Fragment>
  );
};

const inputInputTagUpload = css`
  display: flex;
  text-align-last: center;
  flex-direction: column;
`;

export default FileUpload;
