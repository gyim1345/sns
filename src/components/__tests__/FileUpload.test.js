import React from 'react';
import { mount } from 'enzyme';
import FileUpload from '../FileUpload';
import { post, currentUser } from './testjs';

describe('<FileUpload />', () => {
  it('스냅샷 비교', () => {
    const wrapper = mount(
      <FileUpload posting={post} currentUser={currentUser} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
