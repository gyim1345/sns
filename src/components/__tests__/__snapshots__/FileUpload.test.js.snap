// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FileUpload /> 스냅샷 비교 1`] = `
<FileUpload
  currentUser="gibong@gmail.com"
  posting={
    Object {
      "_id": "5e8ec52d9bb4913b5a844423",
      "id": 1,
      "imageUrl": "http://localhost:3000/static/images/1.jpg",
      "like": Array [
        "gibong@gmail.com",
      ],
      "tag": Array [
        "landscape",
      ],
      "title": "posting with id 1",
      "userName": "gibong@gmail.com",
    }
  }
>
  <form
    onSubmit={[Function]}
  >
    <div
      className="custom-file mb-4"
    >
      <input
        className="custom-file-input"
        id="customFile"
        onChange={[Function]}
        type="file"
      />
    </div>
    <EmotionCssPropInternal
      __EMOTION_LABEL_PLEASE_DO_NOT_USE__="FileUpload"
      __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
      css={
        Object {
          "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkZpbGVVcGxvYWQuanMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBc0crQiIsImZpbGUiOiJGaWxlVXBsb2FkLmpzIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0LCB7IEZyYWdtZW50LCB1c2VTdGF0ZSB9IGZyb20gJ3JlYWN0JztcbmltcG9ydCB7IGNzcyB9IGZyb20gJ0BlbW90aW9uL2NvcmUnO1xuXG5pbXBvcnQgTWVzc2FnZSBmcm9tICcuL01lc3NhZ2UnO1xuaW1wb3J0IHsgdXBsb2FkUGljdHVyZSwgdXBsb2FkVXNlckltYWdlIH0gZnJvbSAnLi4vYXBpcy91cGxvYWQnO1xuXG5jb25zdCBGaWxlVXBsb2FkID0gKHsgY3VycmVudFVzZXIsIHBvc3RpbmcsIHNldFBvc3RpbmcsIHVzZXJJbmZvIH0pID0+IHtcbiAgY29uc3QgW2ZpbGVzLCBzZXRGaWxlXSA9IHVzZVN0YXRlKCcnKTtcbiAgY29uc3QgW2ZpbGVuYW1lLCBzZXRGaWxlbmFtZV0gPSB1c2VTdGF0ZSgnQ2hvb3NlIEZpbGUnKTtcbiAgY29uc3QgW21lc3NhZ2UsIHNldE1lc3NhZ2VdID0gdXNlU3RhdGUoJycpO1xuICBjb25zdCBbaW1nVVJMLCBzZXRJbWdVUkxdID0gdXNlU3RhdGUoJycpO1xuICBjb25zdCBbaW5wdXQsIHNldElucHV0XSA9IHVzZVN0YXRlKCcnKTtcbiAgY29uc3QgW2lucHV0VGFnLCBzZXRJbnB1dFRhZ10gPSB1c2VTdGF0ZSgnJyk7XG4gIGNvbnN0IG9uSW1hZ2VDaGFuZ2UgPSBldmVudCA9PiB7XG4gICAgaWYgKGV2ZW50LnRhcmdldC5maWxlcyAmJiBldmVudC50YXJnZXQuZmlsZXNbMF0pIHtcbiAgICAgIHNldEltZ1VSTChVUkwuY3JlYXRlT2JqZWN0VVJMKGV2ZW50LnRhcmdldC5maWxlc1swXSkpO1xuICAgIH1cbiAgfTtcblxuICBjb25zdCBvbkNoYW5nZUlucHV0ID0gZSA9PiB7XG4gICAgc2V0SW5wdXQoZS50YXJnZXQudmFsdWUpO1xuICB9O1xuXG4gIGNvbnN0IG9uQ2hhbmdlVGFnSW5wdXQgPSBlID0+IHtcbiAgICBzZXRJbnB1dFRhZyhlLnRhcmdldC52YWx1ZSk7XG4gIH07XG5cbiAgY29uc3Qgb25DaGFuZ2UgPSBlID0+IHtcbiAgICBvbkltYWdlQ2hhbmdlKGUpO1xuICAgIHNldEZpbGUoZS50YXJnZXQuZmlsZXNbMF0pO1xuICAgIHNldEZpbGVuYW1lKGUudGFyZ2V0LmZpbGVzWzBdLm5hbWUpO1xuICB9O1xuICBjb25zb2xlLmxvZyhmaWxlcyk7XG5cbiAgY29uc3Qgb25TdWJtaXQgPSBhc3luYyBlID0+IHtcbiAgICBlLnByZXZlbnREZWZhdWx0KCk7XG4gICAgY29uc3QgZm9ybURhdGEgPSBuZXcgRm9ybURhdGEoKTtcbiAgICBmb3JtRGF0YS5hcHBlbmQoJ2ZpbGVzJywgZmlsZXMpO1xuICAgIGZvcm1EYXRhLmFwcGVuZCgnaW5wdXQnLCBpbnB1dCk7XG4gICAgZm9ybURhdGEuYXBwZW5kKCdpbnB1dFRhZycsIGlucHV0VGFnKTtcbiAgICBmb3JtRGF0YS5hcHBlbmQoJ3VzZXInLCBjdXJyZW50VXNlcik7XG4gICAgaWYgKHBvc3RpbmcpIHtcbiAgICAgIHRyeSB7XG4gICAgICAgIGNvbnN0IGRhdGEgPSBhd2FpdCB1cGxvYWRQaWN0dXJlKGZvcm1EYXRhKTtcbiAgICAgICAgcG9zdGluZyAmJiBzZXRQb3N0aW5nKFsuLi5wb3N0aW5nLCBkYXRhLnBvc3RzXSk7XG4gICAgICAgIHNldE1lc3NhZ2UoJ0ZpbGUgVXBsb2FkZWQnKTtcbiAgICAgIH0gY2F0Y2ggKGVycikge1xuICAgICAgICBpZiAoZXJyLnJlc3BvbnNlLnN0YXR1cyA9PT0gNTAwKSB7XG4gICAgICAgICAgc2V0TWVzc2FnZSgnVGhlcmUgd2FzIGEgcHJvYmxlbSB3aXRoIHRoZSBzZXJ2ZXInKTtcbiAgICAgICAgfSBlbHNlIHtcbiAgICAgICAgICBzZXRNZXNzYWdlKGVyci5yZXNwb25zZS5kYXRhLm1zZyk7XG4gICAgICAgIH1cbiAgICAgIH1cbiAgICB9IGVsc2Uge1xuICAgICAgdHJ5IHtcbiAgICAgICAgY29uc3QgZGF0YSA9IGF3YWl0IHVwbG9hZFVzZXJJbWFnZShmb3JtRGF0YSk7XG4gICAgICAgIHVzZXJJbmZvKCk7XG4gICAgICAgIHNldE1lc3NhZ2UoJ0ZpbGUgVXBsb2FkZWQnKTtcbiAgICAgIH0gY2F0Y2ggKGVycikge1xuICAgICAgICBpZiAoZXJyLnJlc3BvbnNlLnN0YXR1cyA9PT0gNTAwKSB7XG4gICAgICAgICAgc2V0TWVzc2FnZSgnVGhlcmUgd2FzIGEgcHJvYmxlbSB3aXRoIHRoZSBzZXJ2ZXInKTtcbiAgICAgICAgfSBlbHNlIHtcbiAgICAgICAgICBzZXRNZXNzYWdlKGVyci5yZXNwb25zZS5kYXRhLm1zZyk7XG4gICAgICAgIH1cbiAgICAgIH1cbiAgICB9XG4gIH07XG5cbiAgcmV0dXJuIChcbiAgICA8RnJhZ21lbnQ+XG4gICAgICB7bWVzc2FnZSA/IDxNZXNzYWdlIG1zZz17bWVzc2FnZX0gLz4gOiBudWxsfVxuICAgICAgPGZvcm0gb25TdWJtaXQ9e29uU3VibWl0fT5cbiAgICAgICAgPGRpdiBjbGFzc05hbWU9XCJjdXN0b20tZmlsZSBtYi00XCI+XG4gICAgICAgICAgPGlucHV0XG4gICAgICAgICAgICB0eXBlPVwiZmlsZVwiXG4gICAgICAgICAgICBjbGFzc05hbWU9XCJjdXN0b20tZmlsZS1pbnB1dFwiXG4gICAgICAgICAgICBpZD1cImN1c3RvbUZpbGVcIlxuICAgICAgICAgICAgb25DaGFuZ2U9e29uQ2hhbmdlfVxuICAgICAgICAgIC8+XG4gICAgICAgIDwvZGl2PlxuXG4gICAgICAgIHtpbWdVUkwgIT09ICcnICYmIChcbiAgICAgICAgICA8aW1nXG4gICAgICAgICAgICBpZD1cInRhcmdldFwiXG4gICAgICAgICAgICBzcmM9e2ltZ1VSTH1cbiAgICAgICAgICAgIHN0eWxlPXt7IG1heFdpZHRoOiAnNTAwcHgnLCBtYXhIZWlnaHQ6ICc1MDBweCcgfX1cbiAgICAgICAgICAvPlxuICAgICAgICApfVxuICAgICAgICA8ZGl2IGNzcz17W2lucHV0SW5wdXRUYWdVcGxvYWRdfT5cbiAgICAgICAgICA8dGV4dGFyZWEgdHlwZT1cInRleHRcIiBvbkNoYW5nZT17b25DaGFuZ2VJbnB1dH0gcGxhY2Vob2xkZXI9XCJUaXRsZVwiIC8+XG4gICAgICAgICAgPGlucHV0IHR5cGU9XCJ0ZXh0XCIgb25DaGFuZ2U9e29uQ2hhbmdlVGFnSW5wdXR9IHBsYWNlaG9sZGVyPVwiVGFnXCIgLz5cbiAgICAgICAgICA8aW5wdXRcbiAgICAgICAgICAgIHR5cGU9XCJzdWJtaXRcIlxuICAgICAgICAgICAgdmFsdWU9XCJVcGxvYWRcIlxuICAgICAgICAgICAgY2xhc3NOYW1lPVwiYnRuIGJ0bi1wcmltYXJ5IGJ0bi1ibG9jayBtdC00XCJcbiAgICAgICAgICAvPlxuICAgICAgICA8L2Rpdj5cbiAgICAgIDwvZm9ybT5cbiAgICA8L0ZyYWdtZW50PlxuICApO1xufTtcblxuY29uc3QgaW5wdXRJbnB1dFRhZ1VwbG9hZCA9IGNzc2BcbiAgZGlzcGxheTogZmxleDtcbiAgdGV4dC1hbGlnbi1sYXN0OiBjZW50ZXI7XG4gIGZsZXgtZGlyZWN0aW9uOiBjb2x1bW47XG5gO1xuXG5leHBvcnQgZGVmYXVsdCBGaWxlVXBsb2FkO1xuIl19 */",
          "name": "151ay85-inputInputTagUpload-FileUpload",
          "next": undefined,
          "styles": "display:flex;text-align-last:center;flex-direction:column;;label:inputInputTagUpload;;;label:FileUpload;/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkZpbGVVcGxvYWQuanMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBd0ZhIiwiZmlsZSI6IkZpbGVVcGxvYWQuanMiLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgUmVhY3QsIHsgRnJhZ21lbnQsIHVzZVN0YXRlIH0gZnJvbSAncmVhY3QnO1xuaW1wb3J0IHsgY3NzIH0gZnJvbSAnQGVtb3Rpb24vY29yZSc7XG5cbmltcG9ydCBNZXNzYWdlIGZyb20gJy4vTWVzc2FnZSc7XG5pbXBvcnQgeyB1cGxvYWRQaWN0dXJlLCB1cGxvYWRVc2VySW1hZ2UgfSBmcm9tICcuLi9hcGlzL3VwbG9hZCc7XG5cbmNvbnN0IEZpbGVVcGxvYWQgPSAoeyBjdXJyZW50VXNlciwgcG9zdGluZywgc2V0UG9zdGluZywgdXNlckluZm8gfSkgPT4ge1xuICBjb25zdCBbZmlsZXMsIHNldEZpbGVdID0gdXNlU3RhdGUoJycpO1xuICBjb25zdCBbZmlsZW5hbWUsIHNldEZpbGVuYW1lXSA9IHVzZVN0YXRlKCdDaG9vc2UgRmlsZScpO1xuICBjb25zdCBbbWVzc2FnZSwgc2V0TWVzc2FnZV0gPSB1c2VTdGF0ZSgnJyk7XG4gIGNvbnN0IFtpbWdVUkwsIHNldEltZ1VSTF0gPSB1c2VTdGF0ZSgnJyk7XG4gIGNvbnN0IFtpbnB1dCwgc2V0SW5wdXRdID0gdXNlU3RhdGUoJycpO1xuICBjb25zdCBbaW5wdXRUYWcsIHNldElucHV0VGFnXSA9IHVzZVN0YXRlKCcnKTtcbiAgY29uc3Qgb25JbWFnZUNoYW5nZSA9IGV2ZW50ID0+IHtcbiAgICBpZiAoZXZlbnQudGFyZ2V0LmZpbGVzICYmIGV2ZW50LnRhcmdldC5maWxlc1swXSkge1xuICAgICAgc2V0SW1nVVJMKFVSTC5jcmVhdGVPYmplY3RVUkwoZXZlbnQudGFyZ2V0LmZpbGVzWzBdKSk7XG4gICAgfVxuICB9O1xuXG4gIGNvbnN0IG9uQ2hhbmdlSW5wdXQgPSBlID0+IHtcbiAgICBzZXRJbnB1dChlLnRhcmdldC52YWx1ZSk7XG4gIH07XG5cbiAgY29uc3Qgb25DaGFuZ2VUYWdJbnB1dCA9IGUgPT4ge1xuICAgIHNldElucHV0VGFnKGUudGFyZ2V0LnZhbHVlKTtcbiAgfTtcblxuICBjb25zdCBvbkNoYW5nZSA9IGUgPT4ge1xuICAgIG9uSW1hZ2VDaGFuZ2UoZSk7XG4gICAgc2V0RmlsZShlLnRhcmdldC5maWxlc1swXSk7XG4gICAgc2V0RmlsZW5hbWUoZS50YXJnZXQuZmlsZXNbMF0ubmFtZSk7XG4gIH07XG4gIGNvbnNvbGUubG9nKGZpbGVzKTtcblxuICBjb25zdCBvblN1Ym1pdCA9IGFzeW5jIGUgPT4ge1xuICAgIGUucHJldmVudERlZmF1bHQoKTtcbiAgICBjb25zdCBmb3JtRGF0YSA9IG5ldyBGb3JtRGF0YSgpO1xuICAgIGZvcm1EYXRhLmFwcGVuZCgnZmlsZXMnLCBmaWxlcyk7XG4gICAgZm9ybURhdGEuYXBwZW5kKCdpbnB1dCcsIGlucHV0KTtcbiAgICBmb3JtRGF0YS5hcHBlbmQoJ2lucHV0VGFnJywgaW5wdXRUYWcpO1xuICAgIGZvcm1EYXRhLmFwcGVuZCgndXNlcicsIGN1cnJlbnRVc2VyKTtcbiAgICBpZiAocG9zdGluZykge1xuICAgICAgdHJ5IHtcbiAgICAgICAgY29uc3QgZGF0YSA9IGF3YWl0IHVwbG9hZFBpY3R1cmUoZm9ybURhdGEpO1xuICAgICAgICBwb3N0aW5nICYmIHNldFBvc3RpbmcoWy4uLnBvc3RpbmcsIGRhdGEucG9zdHNdKTtcbiAgICAgICAgc2V0TWVzc2FnZSgnRmlsZSBVcGxvYWRlZCcpO1xuICAgICAgfSBjYXRjaCAoZXJyKSB7XG4gICAgICAgIGlmIChlcnIucmVzcG9uc2Uuc3RhdHVzID09PSA1MDApIHtcbiAgICAgICAgICBzZXRNZXNzYWdlKCdUaGVyZSB3YXMgYSBwcm9ibGVtIHdpdGggdGhlIHNlcnZlcicpO1xuICAgICAgICB9IGVsc2Uge1xuICAgICAgICAgIHNldE1lc3NhZ2UoZXJyLnJlc3BvbnNlLmRhdGEubXNnKTtcbiAgICAgICAgfVxuICAgICAgfVxuICAgIH0gZWxzZSB7XG4gICAgICB0cnkge1xuICAgICAgICBjb25zdCBkYXRhID0gYXdhaXQgdXBsb2FkVXNlckltYWdlKGZvcm1EYXRhKTtcbiAgICAgICAgdXNlckluZm8oKTtcbiAgICAgICAgc2V0TWVzc2FnZSgnRmlsZSBVcGxvYWRlZCcpO1xuICAgICAgfSBjYXRjaCAoZXJyKSB7XG4gICAgICAgIGlmIChlcnIucmVzcG9uc2Uuc3RhdHVzID09PSA1MDApIHtcbiAgICAgICAgICBzZXRNZXNzYWdlKCdUaGVyZSB3YXMgYSBwcm9ibGVtIHdpdGggdGhlIHNlcnZlcicpO1xuICAgICAgICB9IGVsc2Uge1xuICAgICAgICAgIHNldE1lc3NhZ2UoZXJyLnJlc3BvbnNlLmRhdGEubXNnKTtcbiAgICAgICAgfVxuICAgICAgfVxuICAgIH1cbiAgfTtcblxuICByZXR1cm4gKFxuICAgIDxGcmFnbWVudD5cbiAgICAgIHttZXNzYWdlID8gPE1lc3NhZ2UgbXNnPXttZXNzYWdlfSAvPiA6IG51bGx9XG4gICAgICA8Zm9ybSBvblN1Ym1pdD17b25TdWJtaXR9PlxuICAgICAgICA8ZGl2IGNsYXNzTmFtZT1cImN1c3RvbS1maWxlIG1iLTRcIj5cbiAgICAgICAgICA8aW5wdXRcbiAgICAgICAgICAgIHR5cGU9XCJmaWxlXCJcbiAgICAgICAgICAgIGNsYXNzTmFtZT1cImN1c3RvbS1maWxlLWlucHV0XCJcbiAgICAgICAgICAgIGlkPVwiY3VzdG9tRmlsZVwiXG4gICAgICAgICAgICBvbkNoYW5nZT17b25DaGFuZ2V9XG4gICAgICAgICAgLz5cbiAgICAgICAgPC9kaXY+XG5cbiAgICAgICAge2ltZ1VSTCAhPT0gJycgJiYgKFxuICAgICAgICAgIDxpbWdcbiAgICAgICAgICAgIGlkPVwidGFyZ2V0XCJcbiAgICAgICAgICAgIHNyYz17aW1nVVJMfVxuICAgICAgICAgICAgc3R5bGU9e3sgbWF4V2lkdGg6ICc1MDBweCcsIG1heEhlaWdodDogJzUwMHB4JyB9fVxuICAgICAgICAgIC8+XG4gICAgICAgICl9XG4gICAgICAgIDxkaXYgY3NzPXtbaW5wdXRJbnB1dFRhZ1VwbG9hZF19PlxuICAgICAgICAgIDx0ZXh0YXJlYSB0eXBlPVwidGV4dFwiIG9uQ2hhbmdlPXtvbkNoYW5nZUlucHV0fSBwbGFjZWhvbGRlcj1cIlRpdGxlXCIgLz5cbiAgICAgICAgICA8aW5wdXQgdHlwZT1cInRleHRcIiBvbkNoYW5nZT17b25DaGFuZ2VUYWdJbnB1dH0gcGxhY2Vob2xkZXI9XCJUYWdcIiAvPlxuICAgICAgICAgIDxpbnB1dFxuICAgICAgICAgICAgdHlwZT1cInN1Ym1pdFwiXG4gICAgICAgICAgICB2YWx1ZT1cIlVwbG9hZFwiXG4gICAgICAgICAgICBjbGFzc05hbWU9XCJidG4gYnRuLXByaW1hcnkgYnRuLWJsb2NrIG10LTRcIlxuICAgICAgICAgIC8+XG4gICAgICAgIDwvZGl2PlxuICAgICAgPC9mb3JtPlxuICAgIDwvRnJhZ21lbnQ+XG4gICk7XG59O1xuXG5jb25zdCBpbnB1dElucHV0VGFnVXBsb2FkID0gY3NzYFxuICBkaXNwbGF5OiBmbGV4O1xuICB0ZXh0LWFsaWduLWxhc3Q6IGNlbnRlcjtcbiAgZmxleC1kaXJlY3Rpb246IGNvbHVtbjtcbmA7XG5cbmV4cG9ydCBkZWZhdWx0IEZpbGVVcGxvYWQ7XG4iXX0= */",
          "toString": [Function],
        }
      }
    >
      <div
        className="css-151ay85-inputInputTagUpload-FileUpload"
      >
        <textarea
          onChange={[Function]}
          placeholder="Title"
          type="text"
        />
        <input
          onChange={[Function]}
          placeholder="Tag"
          type="text"
        />
        <input
          className="btn btn-primary btn-block mt-4"
          type="submit"
          value="Upload"
        />
      </div>
    </EmotionCssPropInternal>
  </form>
</FileUpload>
`;
